deployments/fargate/README.md

```diff
@@ -0,0 +1,137 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, it is assumed that the
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS Task in a **sidecar** container alongside monitored
+applications.
+
+Applies to Collector release v0.30.0 and above which corresponds to image tag 0.30.0 and above
+in the image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+Overview of the container definition for the Collector:
+- Set environment variable `SPLUNK_CONFIG` to the configuration file path. Set `SPLUNK_CONFIG`
+  to the default configuration file path to enable default configuration.
+- Use extension `ecs_observer` in your custom configuration to discover targets.
+- Enable ECS read-only permissions in the task role when using `ecs_observer`.
+- Filter the targets for `ecs_observer` to be within task when the Collector is a sidecar,
+  that is, the Collector and the monitored containers are in the same task.
+- `ecs_observer` is currently limited to Prometheus targets.
+- You can use environment variable `SPLUNK_CONFIG_YAML` instead of `SPLUNK_CONFIG` to specify
+  your configuration YAML directly. No need for a file.
+
+## Default Configuration
+The default configuration file is located at `/etc/otel/collector/fargate_config.yaml`
```

Comment:
Great! Let's ensure getting started uses this by default

Reply:
Are you suggesting that `/etc/otel/collector/fargate_config.yaml` should be used by default without the need to set SPLUNK_CONFIG=`/etc/otel/collector/fargate_config.yaml`?

Reply:
using the env var is fine -- stating that getting started steps and data setup UI steps should have this env var set properly.

Reply:
Included setting SPLUNK_CONFIG in getting started section